include/deal.II/dofs/dof_handler.h

```diff
@@ -1224,6 +1241,31 @@ DoFHandler<dim,spacedim>::get_fe () const
 
 template <int dim, int spacedim>
 inline
+const FiniteElement<dim,spacedim> &
+DoFHandler<dim,spacedim>::get_finite_element
+(const unsigned int) const
```

Comment:
@masterleinad I am not suggesting NOT to write it so! I am saying your function get an argument (default to 0), you should check that it is used only with zeroes inside.